Seissol first example to workflow

arshul opened the pull request.

ivotron committed:
Adds workflow

ivotron committed:
wip

ivotron committed:
Adds seissol action

ivotron committed:
Enables travis

ivotron committed:
More sconstruct tweaks

arshul committed:
download files and execute script

arshul committed:
update seissol executable path

arshul committed:
update seissol executable path

arshul committed:
remove check

arshul committed:
minor fixes

arshul committed:
remove .par files

ivotron commented on workflows/scc18/main.workflow:
```diff
@@ -0,0 +1,12 @@
+  uses = "popperized/zenodo/download@master"
+  env = {
+    ZENODO_RECORD_ID = "439946"
+    ZENODO_OUTPUT_PATH = "./workflows/scc18/zenodo-files"
```
can we rename `zenodo-files` to `input-data/` or something alike?

ivotron commented on workflows/tpv33/main.workflow:
```diff
@@ -0,0 +1,31 @@
+  resolves = "execute"
+}
+
+action "build and test" {
```
let's name this `build` as it is not running tests

ivotron commented on workflows/tpv33/main.workflow:
```diff
@@ -0,0 +1,31 @@
+   "unitTests=fast",
+   "-j8",
+  ]
+}
```
and add another action named `test` that runs unit tests (same as above, but appending `check` to the args list)

arshul committed:
build and test separated

arshul committed:
run seissol in /scc18

arshul committed:
fix script path

arshul committed:
add seissol

arshul committed:
remove submodule

arshul committed:
remove seissol

arshul committed:
add seissol submodule

arshul committed:
seissol fix

arshul committed:
download all files

arshul committed:
decrease cores

arshul committed:
readme added, output added, minor fixes

ivotron commented on workflows/tpv33/scripts/download.sh:
```diff
@@ -0,0 +1,19 @@
+tar -xzf Examples.tar.gz
+cp Examples-master/tpv33/* .
+rm -rf Examples-master Examples.tar.gz
+echo $PWD/Maple/ > DGPATH
```
this has to point to the `seissol/Maple/` folder

ivotron committed:
Adds mods to make tpv33 example work

[skip ci]

arshul committed:
tpv3 primary working

arshul committed:
update FL

ivotron committed:
Tweaks to tpv33 workflow

- adds variables to rely less on hardcoded values
- moves seissol submodule to submodules/ folder
- moves actions/seissol to workflows/tpv33/

ivotron committed:
Removes netcdf from tpv33

ivotron committed:
Changes referenced commit of seissol repo

popper:whitelist[workflows/tpv33/main.workflow]

ivotron committed:
Removes scc18 workflow files

[skip ci]

ivotron committed:
Updates README

[skip ci]
